src/pages/en/guides/environment-variables.md

```diff
@@ -42,11 +44,14 @@ PUBLIC_POKEAPI="https://pokeapi.co/api/v2"
 
 ## Getting environment variables
 
-> In this section we use `[dot]` to mean `.` in `import[dot]meta[dot]env` because of a current bug in our build engine. You should write `.` in your code.
+<p>
```

Comment:
I might have missed this, but why is `<p>` needed here?

Reply:
Inline components break *some* markdown parsing so paragraphs need adding back manually. We shouldn't really love this 😂

Reply:
doh.